File: codeforces/cf1555A_pizzaforces.nim
# vicfred
# https://codeforces.com/contest/1555/problem/A
# basic math
import sequtils, strutils
let t = stdin.readLine.parseInt

for _ in 0..<t:
  let n = stdin.readLine.parseInt
  echo max(6, n + 1) div 2 * 5

File: codeforces/cf1555A_pizzaforces.cpp
// uninhm
// https://codeforces.com/contest/1555/problem/A
// greedy

#include <iostream>

int
main ()
{
  int t;
  std::cin >> t;

  long long int n;
  while (t--)
    {
      std::cin >> n;
      if (n <= 6)
        std::cout << 15 << std::endl;
      else
        std::cout << (n + n%2) * 5 / 2 << std::endl;
    }
}